generators/app/templates/src/app/_app.component.ts

```diff
@@ -29,6 +26,10 @@ import { Angulartics2GoogleAnalytics } from 'angulartics2/ga';
 import { environment } from '@env/environment';
 import { Logger, I18nService } from '@app/core';
 
+<% if (props.target.includes('cordova')) { -%>
+declare var Keyboard: any;
+
```

Comment:
no need to do that see ⬇️

Reply:
ah ok will do :).  I didn't know which option you'd dislike more :P. haha.

Reply:
done :)